package.json

```diff
@@ -116,11 +117,17 @@
       "default": true,
       "order": 5
     },
+    "enableDicy": {
+      "description": "Enable the usage of the [DiCy](https://yitzchak.github.io/dicy/) builder instead of `latexmk`.",
+      "type": "boolean",
```

Comment:
@yitzchak Yeah, I think I like that idea. Unless of course we envision moving entirely to DiCy in the (not too distant) future.